docs/capture_agent_plugins/examples/galicaster_2_manchester/README.md

```diff
@@ -12,6 +12,19 @@ Things to note
 * Figuring out the alsa/pulse audio sources can be difficult depending on your audio hardware
 
 
+# Requirements
+
+With the same user that runs Galicaster (Normally: "galicaster"), you need to install these dependencies:
+
+* python-Meteor
+* pyscreenshot
+* image
+* requests
+
```

Comment:
python-ddp

Reply:
that's a dependency of python-meteor so pip should handle that i think?